backend/models/base.py

```diff
@@ -0,0 +1,63 @@
+"""Base classes and methods for measuring distance or time between points."""
+
+
+class Measurer():
+    """Generic class for measuring distance or time."""
+
+    def __init__(self, api_url=None, api_key=None):
```

Comment:
If the default doesnt do anything then maybe this function should not exist, since there is no real extension of it?

Reply:
Done.